src/frontend/Pages/Rules/Rules.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import Footer from "../../Components/Footer/Footer";
```

Comment:
Just a small suggestion, here you can have the index file inside your component folder and export all your files from there so that you can import here all those components together.
e.g. import {Footer, Header} from "../../Components"

Reply:
Thank you suggestion, I'll do that.